cloudquery/sdk/scheduler/scheduler.py

```diff
@@ -113,6 +113,7 @@ def resolve_table(
                         "failed to resolve resource",
                         client_id=client.id(),
                         table=resolver.table.name,
+                        resource=item,
```

Comment:
Can't this leak sensitive information? Is that something we are alright with?

Reply:
I thought about this, and think the chances of this happening are low enough that we can accept the risk/reward trade-off for now. This will help plugin authors during development, and should be relatively rare in production.

I'm open to other suggestions. Maybe only log it in verbose mode?

Reply:
I think logging it only in debug mode is better, so I've updated it now, PTAL :bow: